src/main/webapp/resources/js/pages/projects/samples/linker/Linker.jsx

```diff
@@ -98,13 +94,22 @@ function Linker() {
             />
           </Form.Item>
         </Form>
-        <CommandText
-          className="t-cmd-text"
-          ellipsis={{ rows: 1 }}
-          copyable={command}
-        >
-          {command}
-        </CommandText>
+        <div
```

Comment:
Make a styled component for this div

Reply:
Updated in c828433cd41c82a34d019bd2377da27bb317f436